Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py

```diff
@@ -104,11 +104,10 @@
 # python particle data access
 ##########################
 
-from pywarpx import _libwarpx, callbacks
 
-_libwarpx.add_real_comp('electrons', 'newPid')
+pywarpx.add_real_comp('electrons', 'newPid')
 
-my_id = _libwarpx.libwarpx.warpx_getMyProc()
+my_id = pywarpx.libwarpx.libwarpx_so.warpx_getMyProc()
```

Comment:
This can now simply be as follows, right?
```suggestion
my_id = pywarpx.getMyProc()
```